src/bridge/test-httpstream.c

```diff
@@ -519,6 +519,12 @@ test_tls_basic (TestTls *test,
   GBytes *bytes;
   GBytes *data;
 
+  if (GLIB_CHECK_VERSION (2, 61, 1) && !GLIB_CHECK_VERSION (2, 61, 2))
```

Comment:
I really dislike that we're checking the glib version for a failure that's known to be in a particular version of glib-networking.  I'm afraid we'll need to do better.

Reply:
Please also add a `HACK` comment here so we can remember to remove it later.